application/src/main/java/com/sanction/lightning/utils/CustomWebRequester.java

```diff
@@ -0,0 +1,13 @@
+package com.sanction.lightning.utils;
+
+import com.restfb.DefaultWebRequestor;
+
+import java.net.HttpURLConnection;
+
+public class CustomWebRequester extends DefaultWebRequestor {
+
+  @Override
+  public void customizeConnection(HttpURLConnection connection) {
+    connection.setChunkedStreamingMode(5242880);
```

Comment:
Magic number here.. you should probably make a `private static final` variable in this class and set it to that value.